pkg/cluster/https.go

```diff
@@ -78,10 +89,12 @@ func (c *Cluster) initClusterAndHTTPS(ctx context.Context) error {
 	return nil
 }
 
+// tlsStorage creates an in-memory cache for dynamiclistener's certificate, backed by a file on disk
+// and the Kubernetes datastore.
 func tlsStorage(ctx context.Context, dataDir string, runtime *config.ControlRuntime) dynamiclistener.TLSStorage {
 	fileStorage := file.New(filepath.Join(dataDir, "tls/dynamic-cert.json"))
 	cache := memory.NewBacked(fileStorage)
 	return kubernetes.New(ctx, func() *core.Factory {
 		return runtime.Core
-	}, "kube-system", ""+version.Program+"-serving", cache)
+	}, "kube-system", version.Program+"-serving", cache)
```

Comment:
This is a nit but any usage of a string literal where there is a constant defined for it is one less hit on a search for usages of said constant (see https://github.com/kubernetes/apimachinery/blob/v0.19.2/pkg/apis/meta/v1/types.go#L292)

Reply:
done! I'll try to replace these as I come across them.